### PR TITLE
Remove duplicate fonts from font-jetbrains-mono

### PR DESCRIPTION
**Pull request by EinJochel**
- Removed font `fonts/variable/JetBrainsMono[wght].ttf`, because it is marked as a duplicate of `fonts/ttf/JetBrainsMono.ttf` (in `Font Book.app`)
- Removed font `fonts/variable/JetBrainsMono-Italic[wght].ttf`, because it is marked as a duplicate of `fonts/ttf/JetBrainsMono-Light.ttf` (in `Font Book.app`)
- Also switched from `appcast` stanza to `livecheck`

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask {{cask_file}}` is error-free.
- [x] `brew style --fix {{cask_file}}` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask-fonts/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask {{cask_file}}` worked successfully.
- [ ] `brew install --cask {{cask_file}}` worked successfully.
- [ ] `brew uninstall --cask {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/font-jetbrains-mono.rb [hunk 1]
```diff
@@ -4,11 +4,15 @@
...
   url "https://github.com/JetBrains/JetBrainsMono/releases/download/v#{version}/JetBrainsMono-#{version}.zip",
       verified: "github.com/JetBrains/JetBrainsMono/"
-  appcast "https://github.com/JetBrains/JetBrainsMono/releases.atom"
   name "JetBrains Mono"
   desc "Typeface made for developers"
   homepage "https://www.jetbrains.com/lp/mono"
 
+  livecheck do
+    url "https://github.com/JetBrains/JetBrainsMono/releases.atom"
+    strategy :sparkle
```
Comment (vitorgalvao) [inline Casks/font-jetbrains-mono.rb:13]: `:gitHub_latest` *if* it’s even needed. Try it without a `livecheck` first.

File: Casks/font-jetbrains-mono.rb [hunk 2]
```diff
@@ -37,6 +41,4 @@
   font "fonts/ttf/JetBrainsMonoNL-Regular.ttf"
   font "fonts/ttf/JetBrainsMonoNL-Thin.ttf"
   font "fonts/ttf/JetBrainsMonoNL-ThinItalic.ttf"
-  font "fonts/variable/JetBrainsMono-Italic[wght].ttf"
```
Comment (vitorgalvao) [inline Casks/font-jetbrains-mono.rb:40]: These are the variable versions. Don’t remove them.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/font-jetbrains-mono.rb (hunk 2 above)
Comment (EinJochel) [reply Casks/font-jetbrains-mono.rb:40]: But those create warnings in the `Font Book.app` and issues in some other apps.

Comment (vitorgalvao) [reply Casks/font-jetbrains-mono.rb:40]: Then nobody is happy. Some people want the variable fonts, others don’t, and we can’t satisfy everyone. For now we’ve decided to go with them, so please restore them.

You can always make a tap without them.

**Comment by vitorgalvao** [comment]
Thank you.

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**